WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -569,6 +569,16 @@
         competition.reload
         expect(competition.results_posted_at.to_f).to be < Time.now.to_f
       end
+
+      it "sends the notification emails to users that competed" do
+        FactoryGirl.create_list(:user_with_wca_id, 4).each do |user|
+          FactoryGirl.create_list(:result, 2, person: user.person, competitionId: competition.id)
+        end
+
+        expect do
+          get :post_results, id: competition
```

Comment:
For completeness, how about also mocking out the call to `notify_users_of_results_presence`? You can see that we do that in our other email tests (https://github.com/jfly/worldcubeassociation.org/blob/bf9f079916608e64d3ddc873da1a932b6bf5a671/WcaOnRails/spec/controllers/registrations_controller_spec.rb#L65).